DiscordChatExporter.Core/Exporting/ExportRequest.cs

```diff
@@ -61,6 +62,7 @@ public partial record ExportRequest
                 "%P" => channel.Category.Position?.ToString() ?? "0",
                 "%a" => after?.ToDate().ToString("yyyy-MM-dd") ?? "",
                 "%b" => before?.ToDate().ToString("yyyy-MM-dd") ?? "",
+                "%d" => DateTime.Now.ToString("yyyy-MM-dd"),
```

Comment:
I will replace this with `DateTimeOffset` for consistency (`SnowFlake.ToDate()` converts to `DateTimeOffset` too).

```suggestion
                "%d" => DateTimeOffset.Now.ToString("yyyy-MM-dd"),
```